wordpress-mobile/WordPress-iOS
# Update Gridicons to 1.0

Updates to use the new 1.0 version of the Gridicons framework. Yes, there are a lot of changes here! I don't expect you to look through every line of code change though :)

**To test**

* Skim through the code changes, pick a few areas and make sure the changes look okay
* Build and run, and do a smoke test through the app checking icons look as you expect

PR submission checklist:
